tests/src/system/basic/Swift3WhiskObjectTests.scala

```diff
@@ -41,18 +41,22 @@ class Swift3WhiskObjectTests
     it should "allow Swift actions to invoke other actions" in withAssetCleaner(wskprops) {
         (wp, assetHelper) =>
             // use Wsk to create action from dat/actions/invokeAction.swift
-            val file = TestUtils.getTestActionFilename("invokeAction.swift")
+            val file = TestUtils.getCatalogFilename("/samples/invoke.swift")
             val actionName = "invokeAction"
             assetHelper.withCleaner(wsk.action, actionName) { (action, _) =>
                 action.create(name = actionName, artifact = Some(file), kind = Some("swift:3"), parameters = Map("key0" -> "value0".toJson), expectedExitCode =0)
             }
 
             // invoke the action
-            val invokeResult = wsk.action.invoke(actionName, Map("key0" -> "value0".toJson), blocking = true, result = false, 0).stdout
-
-            invokeResult should include("It is now")
-            invokeResult shouldNot include("Could not parse date of of the response.")
-            invokeResult shouldNot include("Could not invoke date action.")
+            val run = wsk.action.invoke(actionName, Map("key0" -> "value0".toJson))
+            withActivation(wsk.activation, run) {
+                activation =>
+                    val logs = activation.fields("logs").toString()
```

Comment:
by convention `()` are dropped on methods that are not side-effecting as in `activation.fields("logs").toString`.